### DISCUSSION TITLE
how to switch based on colormode in theme

### DISCUSSION THREAD

**Discussion by hesxenon** [chakra-ui/chakra-ui#4977] [Q&A]
So we've recently upgraded from `1.6.4` to `1.6.10` and noticed that we can no longer `import {mode} from "@chakra-ui/theme-tools"`.

The thing is, we've used this `mode` function to switch between colors in the theme like 
```typescript
baseStyle: (props) => ({
  background: mode("light", "dark")(props)
})
```

So, how do we now choose which color to apply in the theme? Is that no longer an option?

As a sidenote, where did the `Styles` type (which was also exported by theme tools) go?

**Comment by daniel-schulz-codes** [comment]
Hej @hesxenon afaik we're using pseudo-selectors (e.g. `_dark`) nowadays instead of calling the mode function. You can see an example here: https://codesandbox.io/s/holy-framework-izsl3?file=/src/index.tsx

**Reply by hesxenon** [reply]
this seems to work in 90% of the cases, but for `_after` and `_before` elements it doesn't do anything? Should I just use the colorMode that is passed along with the props again?

**Reply by daniel-schulz-codes** [reply]
So in General our mid term goal is to move to those pseudo selectors. Therefor it should work. 

Could you provide an example how _after and _before doesn’t work wir the _dark pseudo selector in codesandbox?
